### DISCUSSION TITLE
Which BT/Zigbee chip option is the better one?

### DISCUSSION THREAD

**Discussion by Helmi** [rbaron/b-parasite#118] [General]
I'm just doing a bit of a research about the bluetooth/zigbee chip.

Which of the two C and E versions of the module using the nRF52833 or nRF52840 chip would be preferable to be used? The E-Version / 52833 seems to be about 10-20% cheaper depending on the source. While most sold versions of the module seem to have an antenna connector instead of an onboard antenna, I found one on Amazon with the antenna on board which is around 1€ cheaper (Amazon Germany) than the C / 52840 version.

I'm just not sure about the differences really. Is any of those version better over the other? Availability seems to be not easy overall with backorders on LCSC as well as still no availability on JLCPCB. Aliexpress on top doesn't seem to be really cheaper than Amazon at least from here.

EDIT: Just found a chip comparison at Nordic: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_nrf52%2Fstruct%2Fnrf52.html As you list both as compatible I guess the half sized Flash and RAM on the 52833 don't really matter as well as the missing CryptoCell.

**Comment by rbaron** [comment]
> As you list both as compatible I guess the half sized Flash and RAM on the 52833 don't really matter as well as the missing CryptoCell.

Yes, for our current use cases they are pretty much the same.
